docs/ml-guide.md

```diff
@@ -951,4 +951,4 @@ model.transform(test)
 {% endhighlight %}
 </div>
 
-</div>
\ No newline at end of file
+</div>
```

Comment:
There was no newline char at the end of the file and vim added one, I can remove it if it's an issue.